potodo/_github.py

```diff
@@ -1,32 +1,48 @@
+import re
 import requests
 
-from subprocess import getoutput
+from typing import Mapping
+from subprocess import check_output
 
 
-def get_repo_url():
-    url = getoutput("git remote get-url --all upstream")
+def get_repo_url() -> str:
+    """Tries to get the repository url from git commands
+    """
+    url: str = check_output(
+        "git remote get-url --all upstream", universal_newlines=True
+    )
     if "fatal" in url:
-        url = getoutput("git remote get-url --all origin")
+        url = check_output("git remote get-url --all origin", universal_newlines=True)
     if "fatal" in url:
+        # If the commands didn't work
         raise ValueError(
```

Comment:
[ValueError](https://docs.python.org/3/library/exceptions.html#ValueError) don't pleases me here, it's better suited for bad arguments.

Reply:
Then what ? `OsError` ?

Reply:
Maybe make your own exception, or use RunetimeError.

Reply:
```suggestion
        raise RuntimeError(
```

Reply:
```suggestion
        raise RuntimeError(
```
Lazy